test/acceptance/display_matching_invocations_alongside_expectations_test.rb

```diff
@@ -1,7 +1,7 @@
 require File.expand_path('../acceptance_test_helper', __FILE__)
 require 'mocha/setup'
 
-class DisplayMatchingInvocationsAlongsideExpectations < Mocha::TestCase
+class DisplayMatchingInvocationsAlongsideExpectationsInVerboseMode < Mocha::TestCase
```

Comment:
Minor: The renaming of the test class and its methods is orthogonal to the stated purpose of this commit ("add test for yielding & returning nothing") and so it would be good to extract it into its own commit. Also it would be good to rename the file to match the class name.

Reply:
It looks as if you've removed the changes to the test class name from 90c57c33 ("add test for yielding & returning nothing") to partially address this. 👍

Reply:
Resolved in https://github.com/freerange/mocha/pull/394#issuecomment-551927383.